_includes/devdoc/guide_transactions.md

```diff
@@ -645,22 +645,16 @@ enable locktime.
 
 {% autocrossref %}
 
-Transactions typically pay transaction fees based on the total byte size
-of the signed transaction.  The transaction fee is given to the
+Transactions pay fees based on the total byte size of the signed transaction. Fees per byte are calculated based on current demand for space in mined blocks with fees rising as demand increases.  The transaction fee is given to the
 Bitcoin miner, as explained in the [block chain section][section block chain], and so it is
 ultimately up to each miner to choose the minimum transaction fee they
 will accept.
 
-<!-- TODO: check: 50 KB or 50 KiB?  Not that transactors care... -->
+Deprecation of high-priority-transactions:
 
-By default, miners reserve 50 KB of each block for [high-priority
-transactions][/en/glossary/high-priority-transaction]{:#term-high-priority-transactions}{:.term} which spend satoshis that haven't been spent for a long
-time.  The remaining space in each block is typically allocated to transactions
-based on their fee per byte, with higher-paying transactions being added
-in sequence until all of the available space is filled.
+Older versions of Bitcoin Core recognized so-called "[high-priority transactions][/en/glossary/high-priority-transaction]{:#term-high-priority-transactions}{:.term}" which spent satoshis which had not moved for a long time.  These transaction were often exempt from the normal fee requirements. Before Bitcoin Core 0.12, 50 KB of each block would be reserved for these high-priority transactions, however this is now set to 0 KB by default. As of Bitcoin Core 0.14, the entire concept of "high priority" / free transactions will be removed, meaning that all transactions will simply be prioritized based on their fee per byte, with higher-paying transactions being added in sequence until all of the available space is filled.
```

Comment:
version .14 will have relay disabled by default. (You can still enable most of the priority logic in 0.14)

Removal is targeted for 0.15.